lib/model/connection.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * connection.js
```

Comment:
Since it's going to happen so soon I'd say just wait and then merge develop in here and this stuff should go away. Definitely hold for that though.